internal/plugins/openshift/v1/plugin.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2021 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1
+
+import (
+	"fmt"
+	"path/filepath"
+	"regexp"
+
+	"github.com/spf13/afero"
+	"sigs.k8s.io/kubebuilder/v2/pkg/model/config"
+	"sigs.k8s.io/kubebuilder/v2/pkg/plugin"
+
+	"github.com/operator-framework/operator-sdk/internal/plugins"
+)
+
+const (
+	pluginName = "openshift" + plugins.DefaultNameQualifier
+
+	// The version of the current OCP release.
+	ocpProductVersion = "4.8"
+	// The version of the latest ubi8/ubi-minimal image build.
+	ubiMinimalVersion = "8.3-230"
+)
+
+var (
+	pluginVersion   = plugin.Version{Number: 1}
+	pluginConfigKey = plugin.Key(pluginName, pluginVersion.String())
+)
+
+// Config configures this plugin, and is saved in the project config file.
+type Config struct{}
+
+// RunInit updates a newly initialized project with OpenShift-specific configuration.
+func RunInit(cfg *config.Config) error {
+
+	if err := replaceImages(afero.NewOsFs()); err != nil {
+		return err
+	}
+
+	// Update the plugin config section with this plugin's configuration.
+	pCfg := Config{}
+	if err := cfg.EncodePluginConfig(pluginConfigKey, pCfg); err != nil {
+		return fmt.Errorf("error writing plugin config for %s: %v", pluginConfigKey, err)
+	}
+
+	return nil
+}
+
+type substitution struct {
+	fromTagRE *regexp.Regexp
+	toTag     string
+}
+
+// imageSubstitutions is a map of paths to image substitutions.
+var imageSubstitutions = map[string][]substitution{
+	filepath.Join("config", "default", "manager_auth_proxy_patch.yaml"): []substitution{
+		{
+			regexp.MustCompile(`gcr.io/kubebuilder/kube-rbac-proxy:[^ \n]+`),
+			"registry.redhat.io/openshift4/ose-kube-rbac-proxy:v" + ocpProductVersion,
+		},
+	},
+	filepath.Join("Dockerfile"): []substitution{
+		// Ansible
+		{
+			regexp.MustCompile(`quay.io/operator-framework/ansible-operator:[^ \n]+`),
+			"registry.redhat.io/openshift4/ose-ansible-operator:v" + ocpProductVersion,
+		},
+		// Helm
+		{
+			regexp.MustCompile(`quay.io/operator-framework/helm-operator:[^ \n]+`),
+			"registry.redhat.io/openshift4/ose-helm-operator:v" + ocpProductVersion,
```

Comment:
I do not think that will work for the CI. @jmrodri Have we the images when we do the PR?
For the CI, we might need to replace it again.